apps/onboarding/src/dto/DistributedBlindedPepperDto.ts

```diff
@@ -6,9 +6,8 @@ import {
 } from 'class-validator'
 
 export class DistributedBlindedPepperDto {
-  @IsNotEmpty()
-  @IsPhoneNumber('ZZ')
-  e164Number: string
+  @IsString()
```

Comment:
Not sure about the format of this, but there's also an `@IsHexadecimal()` if that helps.

Reply:
Good call out.. it should be base64 encoded and we have check for that `@IsBase64()`